english Subtitel für ZAP Chat Video 01-08

8linkz opened the pull request:
Video URL: 
Video 01: https://www.youtube.com/watch?v=qrAJ32rdi14
Video 02: https://www.youtube.com/watch?v=RCi9W77bGpI
Video 03: https://www.youtube.com/watch?v=cKG4UWa082A
Video 04: https://www.youtube.com/watch?v=9RbmkT9rG5w
Video 05: https://www.youtube.com/watch?v=Rq_d7OLmMfw
Video 06: https://www.youtube.com/watch?v=PnCbIAnauD8
Video 07: https://www.youtube.com/watch?v=19Rptj2be1Y
Video 08: https://www.youtube.com/watch?v=1fcpU54N-mA

Update 18.12.2023:
This file had a lot of hallucinations and I had to make a lot of manual adjustments. At 12:22->12:35 I'm not sure if what's said matches the subtitle.

If anyone else can double check - would be cool.

Update 11.12.2023:
I'm not good with English spelling (commas etc) and Simon sometimes makes long sentences so it's sometimes hard for me to find an "end" to put the point. But that's up to me, so I can't give any guarantees as far as punctuation is concerned. This part would have to be checked by a native speaker or english pro.

I only deleted AI hallucinations, "um" and "uh".

_____________________________________________________
Goal: The goal is that by making the subtitles as good as possible, it makes it easier to translate into other languages - also using AI, for example. This makes the videos accessible to a larger group of people.

How did I create them?
First I downloaded the video and then removed the video track. I transcribed the .mp3 file with the language via Whisper (https://github.com/openai/whisper). Afterwards, I listened to the video to recognize differences and then adjusted them.

Empty parts are ignored by YouTube and I did not remove them (as there is an enumeration here).

8linkz committed:
english Subtitel für ZAP Chat Video 01

Video URL: https://www.youtube.com/watch?v=qrAJ32rdi14&t


Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
english Subtitel für ZAP Chat Video 02

https://www.youtube.com/watch?v=RCi9W77bGpI

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
english Subtitel für ZAP Chat Video 03

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
english Subtitel für ZAP Chat Video 04

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
english Subtitel für ZAP Chat Video 05

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
english Subtitel für ZAP Chat Video 06

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_05_Modern_Apps_Part_1_english.txt to ZAP_Chat_05_Modern_Apps_Part_1_english.srt

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_04_ZAPit_english.txt to ZAP_Chat_04_ZAPit_english.srt

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_03_Break_Points_english.txt to ZAP_Chat_03_Break_Points_english.srt

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_02_Authentication_Tester_english.txt to ZAP_Chat_02_Authentication_Tester_english.srt

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_01_Introduction_english.txt to ZAP_Chat_01_Introduction_english.srt

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

psiinon commented on ZAP_Chat_04_ZAPit_english.srt:
```diff
@@ -0,0 +1,1076 @@
+
+4
+00:00:19,680 --> 00:00:22,640
+And this time I'm actually joined by Yanis.
```
Yiannis as per https://www.zaproxy.org/docs/team/yiannis/

8linkz replied:
I went through every file again, replaced it, should fit.

psiinon commented on ZAP_Chat_04_ZAPit_english.srt:
```diff
@@ -0,0 +1,1076 @@
+
+146
+00:08:38,520 --> 00:08:45,860
+right number of l's. So Simon while you do that tell us a little bit about the wappanalyzer
```
wappalyzer : https://www.zaproxy.org/docs/desktop/addons/technology-detection/

8linkz replied:
I went through every file again, replaced it, should fit.

8linkz committed:
Update ZAP_Chat_04_ZAPit_english.srt

Yanis -> Yannis
zapit -> ZAPit (except commands)

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update ZAP_Chat_05_Modern_Apps_Part_1_english.srt

Yanis -> Yiannis
zapit -> ZAPit

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update ZAP_Chat_06_Automation_Introduction_english.srt

Yanis -> Yiannis
zapit -> ZAPit

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update ZAP_Chat_04_ZAPit_english.srt

wappanalyzer -> wappalyzer

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

psiinon commented:
Many thanks!

psiinon commented:
@thc202 @kingthorin either of you want to check this?
I havnt checked the times or that its word for word correct, just that it looks sane 😉

thc202 commented:
Happy to take a look.

IMO this should be added under a directory not to the top level of the repo (e.g. `subtitles/ZAP Chat/`).

thc202 commented on ZAP_Chat_04_ZAPit_english.srt:
```diff
@@ -0,0 +1,1076 @@
+
+26
+00:01:38,880 --> 00:01:41,680
+Just keeping it short, my name is Yiannis Papouasoglou.
```
Pavlosoglou

8linkz replied:
done

thc202 commented on ZAP_Chat_06_Automation_Introduction_english.srt:
```diff
@@ -0,0 +1,1765 @@
+
+84
+00:07:56,780 --> 00:08:03,180
+But as Yannis said it uses traditional spider there's no option for the AJAX spider and no
```
Yiannis

8linkz replied:
done

thc202 commented on ZAP_Chat_06_Automation_Introduction_english.srt:
```diff
@@ -0,0 +1,1765 @@
+
+120
+00:11:49,180 --> 00:11:56,700
+if you don't mind just showing us that? It was actually there so zapbaseline.py so it's a python
```
zap-baseline.py

8linkz replied:
done

thc202 commented on ZAP_Chat_06_Automation_Introduction_english.srt:
```diff
@@ -0,0 +1,1765 @@
+
+154
+00:15:12,480 --> 00:15:16,160
+before you get into the more advanced stuff that simon's about to show us. Yeah and one thing we've
```
Simon

8linkz replied:
done

thc202 commented on ZAP_Chat_04_ZAPit_english.srt:
```diff
@@ -0,0 +1,1076 @@
+
+187
+00:12:43,680 --> 00:12:48,940
+video to be functioning right so going back to simon's quote it's not about taking a medium risk
```
Simon

8linkz replied:
done

thc202 commented on ZAP_Chat_04_ZAPit_english.srt:
```diff
@@ -0,0 +1,1076 @@
+
+221
+00:15:41,800 --> 00:15:45,120
+i can see http/3 is beeing used as a technology
```
being

kingthorin replied:
Is it really 3?

thc202 replied:
Yes.

kingthorin replied:
Okay, thanks

8linkz replied:
done

8linkz replied:
![grafik](https://github.com/zaproxy/zap-media-resources/assets/90318259/fd0746fb-1d88-4418-9705-e8464760b91a)

8linkz replied:
in the lower third under "Technology
And google translate http3 into htb3^^

thc202 commented on ZAP_Chat_04_ZAPit_english.srt:
```diff
@@ -0,0 +1,1076 @@
+
+235
+00:17:01,720 --> 00:17:06,460
+company kyberna and one of the services of course that we had to offer right was pentesting, because
```
Kiberna

8linkz replied:
done

thc202 commented on ZAP_Chat_06_Automation_Introduction_english.srt:
```diff
@@ -0,0 +1,1765 @@
+
+51
+00:05:01,120 --> 00:05:08,280
+it's made one request and it got a 200. I haven't actually got the wappalizer add-on installed so
```
wappalyzer

8linkz replied:
done Mh I think I shouldn't review this at 05:30 AM without coffee ~.-

8linkz committed:
Update ZAP_Chat_04_ZAPit_english.srt

Papouasoglou -> Pavlosoglou
simon's -> Simon's
beeing -> being
kyberna -> Kiberna

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update ZAP_Chat_06_Automation_Introduction_english.srt

Yannis -> Yiannis
zapbaseline.py -> zap-baseline.py
simon's -> Simon's
wappalizer -> wappalyzer

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
english Subtitel für ZAP Chat Video 07

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update ZAP_Chat_07_Automation_Framework_Part_1_english.srt

purge the "um"

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

thc202 commented on ZAP_Chat_07_Automation_Framework_Part_1_english.srt:
```diff
@@ -64,7 +64,7 @@ Great. And I'll move some of the toolbars out of my way so I can see what's goin
 17
 00:01:13,640 --> 00:01:18,180
-So what I wanted to show you is start with if we go to documentation.
+So what I wanted to show you is start with if we go to docentation.
```
Removed um in other words.

8linkz replied:
done

8linkz replied:
so now it says documentation again

thc202 replied:
There were several words not just this one: https://github.com/zaproxy/zap-media-resources/pull/4/commits/4c63d3b0639d12f238ad3db92e0709689d3171e6

8linkz replied:
ok no idea why this happened now -.- I'll look over it again later

Edit: only "ducumentation" should be affected, I must have clicked wrong...adjusted the rest...

8linkz committed:
Update ZAP_Chat_07_Automation_Framework_Part_1_english.srt

fix the um

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update ZAP_Chat_07_Automation_Framework_Part_1_english.srt

add ums

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
english Subtitel für ZAP Chat Video 08

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz commented:
add .srt file for Video 08.

psiinon commented:
@8linkz did you see this?

> 
> IMO this should be added under a directory not to the top level of the repo (e.g. `subtitles/ZAP Chat/`).

Also, would be good to open new PRs for new translations otherwise we'll never finish reviewing this one 😉

kingthorin commented:
Let's not add anything more to this PR (after 8) so that we can get it done and merged.

8linkz committed:
Create subtitles

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Delete subtitles

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Create test

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_01_Introduction_english.srt to subtitles/ZAP Chat/ZAP_Chat_01_Introduction_english.srt

move file

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_02_Authentication_Tester_english.srt to subtitles/ZAP Chat/ZAP_Chat_02_Authentication_Tester_english.srt

move file

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_03_Break_Points_english.srt to subtitles/ZAP Chat/ZAP_Chat_03_Break_Points_english.srt

move file

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_04_ZAPit_english.srt to subtitles/ZAP Chat/ZAP_Chat_04_ZAPit_english.srt

move file

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_05_Modern_Apps_Part_1_english.srt to subtitles/ZAP Chat/ZAP_Chat_05_Modern_Apps_Part_1_english.srt

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_06_Automation_Introduction_english.srt to subtitles/ZAP Chat/ZAP_Chat_06_Automation_Introduction_english.srt

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_07_Automation_Framework_Part_1_english.srt to subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update and rename ZAP_Chat_08_Automation_Framework_Part_2_english.srt to subtitles/ZAP Chat/ZAP_Chat_08_Automation_Framework_Part_2_english.srt

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Delete subtitles/ZAP Chat/test

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz commented:
> @8linkz did you see this?
> 
> > IMO this should be added under a directory not to the top level of the repo (e.g. `subtitles/ZAP Chat/`).
> 
> Also, would be good to open new PRs for new translations otherwise we'll never finish reviewing this one 😉

Whoever invented the idea of having to create a folder like this (in github) should be tarred and feathered - should be done now.

psiinon reviewed (approved):
LGTM!

thc202 commented on subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt:
```diff
@@ -0,0 +1,1304 @@
+
+97
+00:07:16,320 --> 00:07:22,080
+And at some point I'll try and make these colns better size automatically.
```
`colns` → `columns`

8linkz replied:
done

thc202 commented on subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt:
```diff
@@ -0,0 +1,1304 @@
+
+100
+00:07:23,440 --> 00:07:26,200
+So just for argent's sake, Simon, let's just say that.
```
`argent` → `argument`

8linkz replied:
done

thc202 commented on subtitles/ZAP Chat/ZAP_Chat_07_Automation_Framework_Part_1_english.srt:
```diff
@@ -0,0 +1,1304 @@
+
+166
+00:12:54,100 --> 00:12:59,980
+the threshold, which is off, low, medi, or high to go very fine rate.
```
`medi` → `medium`

8linkz replied:
done

thc202 commented on subtitles/ZAP Chat/ZAP_Chat_08_Automation_Framework_Part_2_english.srt:
```diff
@@ -0,0 +1,1116 @@
+
+221
+00:18:45,780 --> 00:18:49,200
+type of jobs. Excactly. So
```
`Exactly`

8linkz replied:
done

8linkz committed:
Update ZAP_Chat_07_Automation_Framework_Part_1_english.srt

fix issue mentioned by thx202

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

8linkz committed:
Update ZAP_Chat_08_Automation_Framework_Part_2_english.srt

fix issue mentioned by thc202

Signed-off-by: 8linkz <90318259+8linkz@users.noreply.github.com>

thc202 commented:
Thank you!
